### PR TITLE
docs: fix up sphinx tabs 🌚

### PR DESCRIPTION
**Pull request by kgriffs**
Tested on Firefox, Brave, Safari. Also Brave w/ Dark Reader.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -400,7 +400,7 @@ commands =
 basepython = python3.8
 deps = -r{toxinidir}/requirements/docs
 commands =
-    sphinx-build -W -E -b html docs docs/_build/html []
+    sphinx-build -j auto -W -E -b html docs docs/_build/html []
```
Comment (kgriffs) [inline tox.ini:403]: Shaved 10s off my local build time, so thought I'd just leave it in.

File: requirements/docs [hunk 2]
```diff
@@ -6,5 +6,4 @@ pygments
 pygments-style-github
 sphinx
 sphinx_rtd_theme
-# NOTE(vytas): pin until we figure out how to deal with the dark mode.
```
Comment (kgriffs) [inline requirements/docs:9]: fixed upstream

**Review by vytas7 — APPROVED** [review]
Looks really good :100:

@CaselIT could you also give this a sanity check on your browsers/settings?

File: tox.ini (hunk 1 above)
Comment (CaselIT) [reply tox.ini:403]: I usually use the makefile

**Review by CaselIT — APPROVED** [review]
looks fine also to me

**Pull request merged**